app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SponsorsFragment.java

```diff
@@ -81,7 +78,11 @@ private void initView() {
         binding.recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
     }
 
-    private static class SponsorAdapter
```

Comment:
Ah, sorry... I misunderstood 🙇 
Because this static class can call `getContext()`, but the method is defined in `ArrayRecyclerAdapter`.
As you said, it's impossible to call non-static method from static class.
Thanks for explanation